internal/pkg/usage/usage.go

```diff
@@ -50,6 +50,22 @@ Common Options:
     -h, --help                      Show this message
 `
 
+var securityProxySetupUsageStr = `
+Usage: %s [options]
+Server Options:	
+    -p, --profile <name>            Indicate configuration profile other than default
+    -r, --registry                  Indicates service should use Registry
+	--insureskipverify=true/false   Indicates if skipping the server side SSL cert verifcation, similar to -k of curl
```

Comment:
name unclear

Reply:
It matches what Go calls the feature of not validating HTTPS host names versus the certificate, but agreed it's not a great name. I think it's fine though, and we also have to support what the 1.0 version of security-api-gateway had so we can't change the name of the option